modules/bibdocfile/lib/bibdocfile.py

```diff
@@ -1564,6 +1571,34 @@ def get_text(self, extract_text_if_necessary=True):
 
         return " ".join(texts)
 
+    def stream_archive_of_latest_files(self, req, format=''):
```

Comment:
The symbol `format` has a precise meaning in the `bibdocfile` context.  Did you mean "small/medium/large" here?  If so, then the function parameter may rather be called differently than `format`... e.g. in `getfile()` you call these `filesizes`...